src/connectors/mnet.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+Connector.playerSelector = '#wrap';
+
+Connector.artistSelector = '#tabArea .ItemTitle.selected .artist';
+
+Connector.trackSelector = '#tabArea .ItemTitle.selected .titsong';
+
+Connector.currentTimeSelector = '#player .progressControl .timeStart';
+
+Connector.durationSelector = '#player .progressControl .timeEnd';
+
+Connector.isPlaying = () => {
+	return $('.playControl .btnPause').length !== 0;
+};
+
+Connector.getTrackArt = () => {
+	return `${$('#player .albumArea .albumImg').find('img').attr('src')}`;
```

Comment:
This could be simplified using the trackArtSelector and using `#player .albumArea .albumImg img`

Reply:
Okay I'll try it. Thanks you :)

Reply:
I added the trackArtSelector code.
Please check my commit :)